### DISCUSSION TITLE
chia blockchain 1.2.11 stopped syncing !!!

### DISCUSSION THREAD

**Discussion by AleksandrPDA** [Chia-Network/chia-blockchain#9258] [Support]
at an altitude of 1071254, the chia blockchain synchronization suddenly stopped, I went through many ways to fix the error, as a result, I deleted the chia blockchain database, and the synchronization went successfully, but it stopped at an altitude of 552032, I re-delete the blockchain database, the synchronization starts and again stops exactly at 552032. Help me solve my problem

**Comment by rmica** [comment]
First thing: restart the node without deleting the blockchain, it'll load the db and continue the sync from there.

**Reply by AleksandrPDA** [reply]
thanks for your help! I was restarting the Chia blockchain, I even did it by restarting the computer. But synchronization does not continue and stands at 552032.

**Reply by Jangir92** [reply]
Hello Aleksandr. please tell me you managed to cope with your problem. sync started?
Здравствуйте, Александр. скажите, пожалуйста, вам удалось справиться со своей проблемой. синхронизация началась?
I have exactly the same problem as you                   
У меня точно такая же проблема, как и у вас

**Comment by AleksandrPDA** [comment]
здравствуйте. Нет ,пока ничего не получается, как видите тут на мой запрос тоже пока никто ничего не может подсказать
Hello. No, so far nothing is working, as you can see here on my request, too, so far no one can tell anything

**Comment by zakura-lab** [comment]
доброго та же проблема 1.2.11 
я видел выписку из лога :типо у вас медлено работает база даных  поставил твердотельник -- побыстрее пошла синхранизация
тут зависит от высоты сети и сетевого пространства 
почаще смотрите логи

**Reply by AleksandrPDA** [reply]
Благодарю за помощь!! А где можно посмотреть логи, а то я не силен в этом?

**Reply by Jangir92** [reply]
Доброго времени суток. Да мне тоже очень интересно как решить нашу общую проблему. Подскажите пожалуйста как это сделать по порядку.

**Reply by Jangir92** [reply]
вот что нашёл в интернете.http://netler.ru/ikt/windows10-events.htm может кому поможет. честно сказать сам пока не разобрался.

**Comment by AleksandrPDA** [comment]
Hello! My problem is not solved !!! Why isn't the support service responding to my request? Can you help me?

**Comment by AleksandrPDA** [comment]
???

**Reply by Jangir92** [reply]
Здраствуйте.я установил старую версию 1.1.6. после удалил старую синхранизацию фаил "db" сейчас жду новой синхронизации. как пройдёт мой пик синхранизации то расскажу получилось или нет.

**Comment by Jangir92** [comment]
Здраствуйте.я установил старую версию 1.1.6 . после удалил старую синхранизацию фаил "db" сейчас жду новой синхронизации. как пройдёт мой пик синхранизации то расскажу получилось или нет.

**Reply by AleksandrPDA** [reply]
Благодарю! будем ждать новостей

**Reply by Jangir92** [reply]
Дорогие пользователи chia block chain.у меня полностью синхронизировалась chia.устанавливаете старую версию 1.1.6 и удаляете папку db. после начинается синхронизация. если вдруг она заканчивается можете установить 1.1.7-1.2.0 у меня сейсас на этой версии всё работает. Всем удачи и всего хорошего. Рад был всем помочь.

**Reply by AleksandrPDA** [reply]
Благодарю Джангир! В одном своем посте я тебе  написал что у меня тоже все заработало. После 1.1.6 я поставил 1.2.7.
